### PR TITLE
Adds asymmetric coat, blouses, flags, and Assunzione items to the loadout.

### PR DESCRIPTION
**Pull request by TheStryker**
This PR adds a Fisanduhian flag to atone for my Dominia PRs. Also adds a Gadpathurian flag. Flavor texts by Schwann.
image
image
Also adds an Assunzione warding sphere and sheath with minimal glow to the religion tab in the loadout. It shatters upon impact.
image
image
image
Adds an asymmetric coat as seen in the previous pictures to the suits loadout tab.
image
Adds puffy and long-sleeved blouses to the loadout (accessories/shirts tab).
image

### CODE REVIEW

**Comment by TheStryker** [comment]
!review

**Comment by TheStryker** [comment]
Adds an Assunzione amulet to the Loadout Religion Tab, courtesy of Pretzel.
image

File: code/game/objects/items/weapons/chaplain_items.dm [hunk 1]
```diff
@@ -193,3 +193,58 @@
...
+	storage_slots = 1
+	drop_sound = 'sound/items/drop/axe.ogg'
+	pickup_sound = 'sound/items/pickup/axe.ogg'
+
+/obj/item/storage/assunzionesheath/update_icon()
+	if(contents.len)
+		icon_state = "assunzionesheath"
+	else
+		icon_state = "assunzionesheath_empty"
+	return
```
Comment (Geevies) [inline code/game/objects/items/weapons/chaplain_items.dm:248]:
```suggestion
```

Comment (Geevies) [reply code/game/objects/items/weapons/chaplain_items.dm:248]: lotsa empty whitespace beneath that too

**Review by Geevies — APPROVED** [review]

File: code/game/objects/items/weapons/chaplain_items.dm [hunk 2]
```diff
@@ -193,3 +193,58 @@
...
+	light_range = 1.4
+	light_power = 1.4
+	light_color = LIGHT_COLOR_BLUE
+	w_class = ITEMSIZE_SMALL
+	drop_sound = 'sound/items/drop/glass.ogg'
+	pickup_sound = 'sound/items/pickup/glass.ogg'
+
+/obj/item/assunzioneorb/proc/shatter()
+	visible_message(SPAN_WARNING("\The [src] shatters!"), SPAN_WARNING("You hear a small glass object shatter!"))
+	force = 5
```
Comment (Alberyk) [inline code/game/objects/items/weapons/chaplain_items.dm:215]: What is the purpose of doing this if the item will be deleted?

File: code/game/objects/items/weapons/chaplain_items.dm [hunk 3]
```diff
@@ -193,3 +193,58 @@
...
+	sharp = TRUE
```
Comment (Alberyk) [inline code/game/objects/items/weapons/chaplain_items.dm:216]: What is the purpose of doing this if the item will be deleted?

**Review by Alberyk — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/chaplain_items.dm (hunk 3 above)
Comment (TheStryker) [reply code/game/objects/items/weapons/chaplain_items.dm:216]: It's there so that there's a prompt that says it broke, similar to other glass items.

**Review by Geevies — APPROVED** [review]

File: code/game/objects/items/weapons/chaplain_items.dm (hunk 3 above)
Comment (Alberyk) [reply code/game/objects/items/weapons/chaplain_items.dm:216]: This var is pointless, remove it.

File: code/game/objects/items/weapons/chaplain_items.dm (hunk 2 above)
Comment (Alberyk) [reply code/game/objects/items/weapons/chaplain_items.dm:215]: This var is pointless, remove it.

File: code/game/objects/items/weapons/chaplain_items.dm (hunk 3 above)
Comment (TheStryker) [reply code/game/objects/items/weapons/chaplain_items.dm]: force and sharp have been removed.
I misunderstood the code and thought sharp made it cause bleeding. As for the force, I transferred it to the item itself.

**Review by Alberyk — APPROVED** [review]

**Pull request merged**